Source/CarthageKit/Dependency.swift

```diff
@@ -148,7 +148,10 @@ extension Dependency: Scannable {
 					if url.scheme == "https" || url.scheme == "file" {
 						return .success(self.binary(BinaryURL(url: url, resolvedDescription: url.description)))
 					} else if url.scheme == nil {
-						let absoluteURL = URL(fileURLWithPath: url.relativePath, isDirectory: false, relativeTo: base).standardizedFileURL
+						// This can use URL.init(fileURLWithPath:isDirectory:relativeTo:) once we can target 10.11+
+						let absoluteURL = url.relativePath
+							.withCString { URL(fileURLWithFileSystemRepresentation: $0, isDirectory: false, relativeTo: base) }
+							.standardizedFileURL
```

Comment:
This should be equivalent. https://github.com/apple/swift-corelibs-foundation/blob/master/Foundation/NSURL.swift#L359-L363